From 86408fa5c3606e911b013f855d2396059ab3317a Mon Sep 17 00:00:00 2001
From: Thomas Leonard <talex5@gmail.com>
Date: Mon, 27 Aug 2012 11:27:37 +0100
Subject: [PATCH] Only install runenv.cli.template on NT systems

---
 setup.py |    5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 6556c2f..6bc1880 100644
--- a/setup.py
+++ b/setup.py
@@ -54,9 +54,12 @@ class build_with_data(build_py):
 		"zeroinstall/gtkui/desktop.ui",
 		"zeroinstall/gtkui/cache.ui",
 		"zeroinstall/injector/EquifaxSecureCA.crt",
-		"zeroinstall/injector/runenv.cli.template",
 		"zeroinstall/zerostore/_unlzma",
 	]
+
+	if os.name == "nt":
+		package_data_files.append("zeroinstall/injector/runenv.cli.template")
+
 	def run(self):
 		# Copy .py files and build, as usual
 		build_py.run(self)
-- 
1.7.10.4

